.changes/unreleased/Added-20240806-231920.yaml

```diff
@@ -0,0 +1,6 @@
+kind: Added
+body: Add `Digest` method to `File`
```

Comment:
:eyes: can we get another line in the changelog body here explaining roughly what this method does and what it's for?

I'd really like to have users read release notes and get excited about the features/fixes we're shipping, and having a bit more of an explanation for each point goes a long way to getting that :tada: :tada:

Reply:
Sure